addons/website/static/src/js/content/menu.js

```diff
@@ -496,7 +499,9 @@ publicWidget.registry.menuDirection = publicWidget.Widget.extend({
      * @override
      */
     start: function () {
-        this.defaultAlignment = this.$el.is('.ms-auto, .ms-auto ~ *') ? 'right' : 'left';
+        this.defaultAlignment = this.$el.is('.ms-auto, .ms-auto ~ *') ? 'end' : 'start';
+        this.rtl = !!this.el.closest('#wrapwrap.o_rtl');
```

Comment:
I don't have any info on this change in the commit message ;) It's a welcome fix but it should be in a previous version ;) As discussed in Discord you can create another PR for this in the right version...